templates/azure/fips.html

```diff
@@ -0,0 +1,405 @@
+              <p>For companies looking to leverage industry benchmarks for hardening, Ubuntu Pro makes two leading profiles available.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">HIPAA, PCI, ISO compliance</h3>
+            <div class="p-matrix__desc">
+              <p>Healthcare, financial services, insurance and other organisations that operate in heavily-regulated industries ensure compliance with HIPAA, PCI and ISO.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Predictable and Secure</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu Pro FIPS is based on <a href="https://ubuntu.com/azure/pro">Ubuntu Pro premium images</a>, with Azure optimisations built in and the most comprehensive open source security coverage in the industry.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Common Criteria EAL2 certification</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu Pro FIPS 16.04 has been evaluated to assurance level EAL2, with Common Criteria EAL2 coming soon for Ubuntu Pro FIPS 18.04  coming soon.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Open source security</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu Pro adds security coverage for the most important open source applications like Apache Kafka, NGINX, MongoDB, Redis and PostgreSQL...</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">10-year lifetime</h3>
+            <div class="p-matrix__desc">
+              <p>Canonical backs Ubuntu Pro FIPS for 10 years*, ensuring security updates are available throughout, with a guaranteed upgrade path.</p>
+            </div>
+          </div>
+        </li>
+        <li class="p-matrix__item">
+          <div class="p-matrix__content">
+            <h3 class="p-matrix__title">Priced for the cloud</h3>
+            <div class="p-matrix__desc">
+              <p>Ubuntu Pro FIPS pricing tracks the underlying compute cost, starting at under $0.01 for the smallest B1ls instance type, and ramping down to less than 1% of hourly compute.</p>
+            </div>
+          </div>
+        </li>
+      </ul>
+      <p id="index-1"><small>* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years.</small></p>
+      <a href="https://ubuntu.com/azure/contact-us" class="p-button--positive">Let's talk Pro</a>
```

Comment:
Referring to the copy doc this should be:

```suggestion
      <a href="https://ubuntu.com/azure/contact-us" class="p-button--positive">Let's talk about Ubuntu FIPS</a>
```

Reply:
Fixed.